### ISSUE TITLE
Option to change wording of '2 day rate'

### ISSUE THREAD

**Issue by browningjp** [Jon-Salmon/Durham-Hires#37]
On the catalogue page the 'day rate' is labelled as '2 day rate'. Not sure if this is a mistake or just how trevs run their hires? Could we add in an option to edit this text? (and have it consistent throughout the system e.g. adding items, invoices etc)

**Comment by Jon-Salmon** [comment]
Should probaboly do that! Might need to adjust the way that costs are calculated too then?

Currently:

**2 day hire**
 - Collect items day 1 after lunch
 - Have event day 2
 - Return items day 3 before lunch

**Week hire**
 - Collect items day 1 after lunch
 - Return items day 8 before lunch

Would, for example, you like it to be configurable to:

**1 day hire**
 - Collect items day 1 after lunch
 - Return items day 2 before lunch

**Comment by browningjp** [comment]
There are no times actually used on the site - do we even need to worry about what time the cut-off is? Just let colleges choose their own in their terms and conditions

**Comment by Jon-Salmon** [comment]
The time isn't made use of from a user point of view, only internaly within the system and the time itself is arbitrary. Just chost 12 noon GMT as it avoids any BST issues.

Do we want to be able to make the 2-day booking into a 1-day booking for example? e.g. booking is collected at some point Saturday and returned Sunday? Bookings of the same kit would then be allowed for return on Saturday and collection on Sunday.

**Comment by browningjp** [comment]
yes, that would be my preference. I had forgotten about this until a "2-day" hire I've just done!
